scripts/ci/docker_upgrade.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+set -e
+
+function error() {
+  echo >&2 "ERROR: ${1} Aborting." && false
+}
+
+function warning() {
+  echo >&2 "WARNING: ${1}"
+}
+
+check_dockerfile_was_changed() {
+  echo "-- check_dockerfile_was_changed"
+
+  [[ $# == 2 ]] || error "Expected exactly 2 parameters: 'check_dockerfile_was_changed <IMAGE_NAME> <IMAGE_VARIANT>'"
```

Comment:
As far as I'm concerned: Just inline everything, this is getting a bit silly :-).

Reply:
And if you just inline everything, you don't need these checks, you don't need the locals you don't need the function definitions and you don't need the calls - the script will get surprisingly short that way ;-).

Reply:
yeah.. I was thinking exactly the same.. I will now just remove the function definitions and the calls